ReleaseChecklist.md

```diff
@@ -7,19 +7,38 @@
  - [ ] DockerHub account with push rights to the [``solc`` image](https://hub.docker.com/r/ethereum/solc).
  - [ ] Lauchpad (Ubuntu One) account with a membership in the ["Ethereum" team](https://launchpad.net/~ethereum) and
        a gnupg key for your email in the ``ethereum.org`` domain (has to be version 1, gpg2 won't work).
+ - [ ] Access to an Ubuntu or Debian system with the following packages installed: `devscripts`, `debhelper`, `dput`, `git`, `wget`, `ca-certificates`.
```

Comment:
That's not true, the required packages are available on all sorts of distros, including e.g. arch.

Reply:
Oh, I didn't know they were packaged on Arch. I see they're in AUR, so I guess that's good enough.

I was planning to add a dockerfile of the docker image I used last time to `scripts`. 
Should I do that anyway or do you think there's no point if you can install these on other distros? It still eliminates some setup (e.g. the `dput.cf` file).

Reply:
I don't think this should suggest to use a docker image, resp. we shouldn't add another script to maintain just to avoid one-time effort for this.

Reply:
ok